connection_util.go

```diff
@@ -92,12 +92,15 @@ func (sc *snowflakeConn) processFileTransfer(
 	query string,
 	isInternal bool) (
 	*execResponse, error) {
+	options := &SnowflakeFileTransferOptions{
+		RaisePutGetError: true,
```

Comment:
I guess the next step would be to remove this parameter altogether? 
I would add a jira task for this.

Reply:
This is a public parameter, do we want to release a BCR?